kairos-deposit-contract/malicious-reader/src/main.rs

```diff
@@ -0,0 +1,12 @@
+#![no_std]
+#![no_main]
+use casper_contract::contract_api::{account, runtime, system};
+use casper_types::{URef, U512};
+
```

Comment:
Please add a comment on what the malicious reader is supposed to do.

Reply:
I will resolve this since I will add comments to all session code as mentioned above. I will keep the other conversation open.